kedro/runner/sequential_runner.py

```diff
@@ -75,21 +75,22 @@ def _run(
 
         for exec_index, node in enumerate(nodes):
             try:
-                run_node(node, catalog, hook_manager, self._is_async, session_id)
+                from kedro.runner.task import Task
```

Comment:
This is needed because I refactored `run_node` in the runner to use `Task` and moved methods to `Task` as well. `run_node` isn't actually needed anymore, but removing it would be a breaking change. I could undo the changes to `runner.py` which removes the import from `Task` and then allows it to be imported inside the runner implementations again. The downside is that we'd have duplicated code in `runner.py` and `task.py`.